librz/core/cmd/cmd_search_rop.c

```diff
@@ -4,726 +4,457 @@
 #include <stddef.h>
```

Comment:
Add your name as Copyright by adding a new `// SPDX-FileCopyrightText: ` on top of the old one.